eZ/Publish/Core/Helper/PreviewLocationProvider.php

```diff
@@ -70,8 +95,10 @@ public function loadMainLocation($contentId)
                 return null;
             }
 
+            // @TODO: Repository needs to add full support for draft Locations to avoid having to do this
```

Comment:
```suggestion
```

Reply:
What about something like:

```suggestion
            // NOTE: Once Repository adds support for draft locations (and draft  location ops), then this can be removed 
```